solutions/usaco-600.mdx

```diff
@@ -2,19 +2,105 @@
 id: usaco-600
 source: USACO Platinum 2016 January
 title: Fort Moo
-author: Kevin Sheng
+author: Kevin Sheng, Ryan Chou
 ---
 
-[Official Analysis](http://www.usaco.org/current/data/sol_fortmoo_platinum_jan16.html)
+[Official Analysis (Java)](http://www.usaco.org/current/data/sol_fortmoo_platinum_jan16.html)
 
+## Solution
+
+### Explanation
+
+By computing the [2D Prefix Sum](/silver/more-prefix-sums/) for the grid, we can evaluate the usability of a frame in $\mathcal{O}(1)$ time.
```

Comment:
How is this so much different than the official?

Reply:
even if it isn't, adding an internal solution does no harm

Reply:
+ it implements using a diff lang